app/views/transactions/categories/dropdowns/_row.html.erb

```diff
@@ -2,7 +2,7 @@
 <% is_selected = category.id === @selected_category&.id %>
 
 <%= content_tag :div, class: ["filterable-item flex justify-between items-center border-none rounded-lg px-2 py-1 group w-full", { "bg-gray-25": is_selected }], data: { filter_name: category.name } do %>
-  <%= button_to transaction_path(@transaction, transaction: { category_id: category.id }), method: :patch, class: "flex w-full items-center gap-1.5 cursor-pointer" do %>
+  <%= button_to transaction_row_path(@transaction, transaction: { category_id: category.id }), method: :patch, data: { turbo_frame: dom_id(@transaction) }, class: "flex w-full items-center gap-1.5 cursor-pointer" do %>
```

Comment:
There may be a better way to do this.  Since this is a nested frame, I decided to target the outermost "row" frame so when the user selects a category, it replaces the entire row.  We could certainly get more surgical with this (via streams), but my intention was to move towards the _simplest_ solution possible, hence the `Transactions::RowsController` in tandem with a `turbo_frame_tag dom_id(@transaction)`